stix_shifter_modules/crowdstrike_logscale/README.md

```diff
@@ -0,0 +1,721 @@
+                    "os_ref": "14",
+                    "x_host_type_number": "3",
+                    "host_type": "Server",
+                    "x_service_provider": "AWS_EC2_V2",
+                    "x_service_account_id": "1234",
+                    "x_status": "normal",
+                    "x_system_manufacturer": "Xen",
+                    "x_system_product_name": "HVM domU"
+                },
+                "10": {
+                    "type": "x-crowdstrike-edr-agent",
+                    "load_flags": "1",
+                    "local_time": "2023-12-21T05:47:27.598Z",
+                    "version": "7.05.17706.0",
+                    "config_id_base": "65994763",
+                    "config_id_build": "17706",
+                    "config_id_platform": "3"
+                },
+                "11": {
+                    "type": "ipv4-addr",
+                    "value": "3.4.5.6"
+                },
+                "12": {
+                    "type": "ipv4-addr",
+                    "value": "6.7.8.9",
+                    "resolves_to_refs": [
+                        "13"
+                    ]
+                },
+                "13": {
+                    "type": "mac-addr",
+                    "value": "12:85:26:67:6c:9"
+                },
+                "14": {
+                    "type": "software",
+                    "x_major_version": "10",
+                    "x_minor_version": "0",
+                    "version": "Windows Server 2022",
+                    "x_id": "0",
+                    "name": "Windows"
+                }
+            },
+            "last_observed": "2023-12-21T10:49:06.000Z",
+            "first_observed": "2023-12-21T10:48:33.774Z",
+            "number_observed": 1
+        }
+    ],
+    "spec_version": "2.0"
+}
+```
+
+### Connector Extension
+- As Logscale doesn't have unified data schema, separate mapping files(from_stix_map.json, to_stix_map.json) needs to be 
+  created for each log source(eg: okta, aws_guardduty) that are newly added to this connector module.
+- The structure of log source data ingest into logscale should be of type JSON .
+- The Json data which has been ingested into logscale should be inserted without new line and should be inserted as raw data.
+- The mapping of list (list of values/list of dictionary) fields in from_stix_map should be mentioned with [\*] suffix. 
+  Example, behaviors[\*].id.  Here 'behaviors' is a list of dictionaries with id as attribute key inside behaviors.
+
+### Recommendations
+
+- For connector usage, it is recommended to maintain logs from single log source per repository in Crowdstrike Logscale. 
+  Example Okta logs could be stored in repository_1 and aws_guardduty logs in repository_2.
+- Make sure, there is no parsing error during ingestion of logs into Crowdstrike Logscale.
+
+### Limitations
+- LIKE,MATCHES, <, >, <=, >= operators are not supported for list of dictionary fields.
```

Comment:
As suggested, detailed explanation of these attribute types has been added in readme file.

Reply:
I probably wasn't clear here. I meant that there is no way to tell from the STIX query format. You'd need to find what each field maps to in the API to know.

For example : 

```
      "x_extension": [
        "behaviors[*].alleged_filetype"
      ],
```

`| List of values fields/Array fields | A list containing one or more values |
 "behaviors_processed": ["123","abc"] |`

maps to a list of values. But you only know that because you mapped the from_stix to the query field and then used that to look into the data/api doc. 

From a user perspective there is no way to know without going through these extra hoops. Which would make the limitations difficult to use.

Reply:
Added the details in Appendix section with which has the list of all API fields and the type of field